src/Chainweb/Version.hs

```diff
@@ -767,6 +767,12 @@ coinV2Upgrade Development cid h
 coinV2Upgrade _ _ 1 = True
 coinV2Upgrade _ _ _ = False
 
+-- | Specify the twenty chain fork block height in one place.
+twentyChainUpgradeBlockHeight :: ChainwebVersion -> BlockHeight
+twentyChainUpgradeBlockHeight Development = 400
+twentyChainUpgradeBlockHeight Mainnet01 = maxBound
+twentyChainUpgradeBlockHeight _ = 2
```

Comment:
Doesn't this test chainweb versions that don't do a transition but use pact, like e.g. `FastTimedCPM`?

Reply:
Actually, I think `twentyChainUpgrade` should be derived from `chainwebGraphs` and this function could be dropped. I seems like an unneeded level of indirection. I think `chainGraphs` is the authoritative source for graph transition block heights.

Reply:
It seems that `twentyChainUpgrade` doesn't depend on the actual height of the graph transition. So, I think, we can just drop this change.